components/MarqueeText/samples/MarqueeText.md

```diff
@@ -0,0 +1,24 @@
+---
+title: MarqueeText
+author: Avid29
+description: A control for scrolling text in a marquee fashion.
+keywords: Marquee, Control, Text
+dev_langs:
+  - csharp
+category: Controls
+subcategory: StatusAndInfo
+discussion-id: 231 
+issue-id: 0
+---
+
+# MarqueeText
+
+For more information about this experiment see:
+
+- Discussion: https://github.com/CommunityToolkit/Labs-Windows/discussions/231
```

Comment:
This can be removed, we have the discussion button that will take users there (using the discussion ID).